cmake/modules/packages/FindOGDI.cmake

```
@@ -12,28 +12,50 @@
 #    OGDI_VERSION
 #
 
-find_path(OGDI_INCLUDE_DIR ecs.h PATH_SUFFIXES ogdi)
+find_package(PkgConfig QUIET)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_OGDI QUIET OGDI)
```

case is sensitive for pkg-config
```suggestion
  pkg_check_modules(PC_OGDI QUIET ogdi)
```